### PR TITLE
Add Lunacy 

### PR DESCRIPTION
**Pull request by Pkcarreno**

### CODE REVIEW

**Comment by ripienaar** [comment]
Thanks, please restrict your edit to just adding the entry and not changing every line :)

**Comment by thispsj** [comment]
BTW Lunacy is not an as-a-service offering. It's available for free on Windows Store/App Store.

File: README.md [hunk 1]
```diff
@@ -8,1159 +8,1160 @@ The scope of this particular list is limited to things that infrastructure devel
...
+- [Mockplus iDoc](https://www.mockplus.com/idoc) - Mockplus iDoc is a powerful design collaboration & handoff tool. Free Plan includes 3 users and 5 projects with all features available.
+- [AllTheFreeStock](https://allthefreestock.com) - a curated list of free stock images, audio and videos.
+- [BoxySVG](https://boxy-svg.com/app) — A free installable Web app for drawing SVGs and exporting in svg,png,jpeg an other formats.
+- [clevebrush.com](https://www.cleverbrush.com/) — Free Graphics Design / Photo Collage App, also they offer paid integration of it as component.
+- [cloudconvert.com](https://cloudconvert.com/) — Convert anything to anything. 208 supported formats including videos to gif.
+- [CodeMyUI](https://codemyui.com) - Handpicked collection of Web Design & UI Inspiration with Code Snippets.
+- [designer.io](https://www.designer.io/) — Design tool for UI, illustrations and more. Has a native app. Free.
+- [figma.com](https://www.figma.com) — Online, collaborative design tool for teams; free tier includes unlimited files and viewers with a max of 2 editors and 3 projects.
+- [Lunacy.com](https://icons8.com/lunacy) — A vector design tool for windows with built-in assets.
```
Comment (thispsj) [inline README.md:938]: Lunacy is not a online service. It's just a native app to use on your local machine.

Comment (ripienaar) [reply README.md:938]: I'd consider some of these to be IDE like so will make an exception, but the PR in general isnt acceptable

Comment (thispsj) [reply README.md:938]: Ya the person has made unacceptable number of edits which aren't needed.

**Pull request closed without merging**